### ISSUE TITLE
[1.10.2] Elevator screens don't show levels when greater than ~65 blocks from base elevator

### ISSUE THREAD

**Issue by praetoras** [McJty/RFTools#605]
I have a very tall tower that I've been using the elevator with. The elevator itself works fine, but the screens (despite the more distant levels having their own screen controller, which is successfully linking to them) will not display the floors when the screens are around 60-70 blocks or greater from the base elevator. As long as they're within that range, they will display the levels in the button module, but beyond that, even screens connected to the same screen controller, they are just blank.

Verified all screens were working, and when switching which elevator was considered the root elevator, the screens in its vicinity worked.

**Comment by McJty** [comment]
Are you sure the screens are in the range of the screen controller? It has a limited range. You might need another screen controller higher up for those screens

**Comment by praetoras** [comment]
Yeah, I have each group of elevator modules connected to their own screen controller, making 3 screen controllers in total, and then checked the controllers each to see how many they were connecting with. The screen controllers are connecting to the screens (6 for the top, 2 for the middle, and 1 for the bottom, making 9 screens total) but if I have the "root" elevator for the buttons up at the top, then only the top 7 will show the buttons. If I have the bottom as the root, only the lower 2 can. The  top 6 can't show anything.

**Comment by McJty** [comment]
Can you show a screenshot of that top screen with your mouse hovering over it?

**Comment by praetoras** [comment]
No problem! Hopefully this is helpful :)

http://imgur.com/a/YfTgp

I created an album trying to demonstrate the issue, including the hovering tooltips. Unfortunately, the tooltips always seem to show the same thing regardless of whether or not they're connected. I included the block levels in the descriptions of the images as well, along with other info.

**Comment by McJty** [comment]
Well note the tooltips of the screen. It says '[NOT CONNECTED]' '[NO POWER]' Did you press 'scan' on the screen controller UI? The range of the screen controller is not that big (only -16/+16 in height)

**Comment by praetoras** [comment]
Correct, I did, and I actually tried redoing the system several times. The screens always say [NOT CONNECTED] and [NO POWER] regardless of being connected and having power (you'll note that despite the tooltip displaying that, the screen is actually showing the floor names and levels, which it wouldn't do if it wasn't connected to the screen controller).

I had double-checked all of the screen controllers: the top controller after scanning was connected to 6 screens, and I detached and rescanned just to make sure, but the problem resumed. The middle screen connector is connected to 2 screens, and the bottom screen has its own dedicated connector, all of which are connected.

Again, the elevator itself works fine. The only issue is that the screens below those block levels won't display beyond a certain distance from the root elevator.

**Comment by McJty** [comment]
Very weird. Never seen this issue before and people are using my elevator + screens already...

**Comment by praetoras** [comment]
Yeah, I'm not too surprised there. I doubt many people would run into this kind of situation. This is the first time I had tried using the elevator on such a large structure. Before I had been using it on a smaller one for just 6 screens on a different world and never had any issue, but I also only needed one screen controller for the whole set.

The bottom elevator is at level 79 and the top is at block level 198, so that's over 100 blocks distance. I'm building in an extreme hills biome and had put a fort on one of the taller mountains, hence the gargantuan distance.

**Comment by McJty** [comment]
Well the issue here is not really the elevator but the screens not functioning correctly. BTW, the screens that are not working don't have any other modules in them by chance?

**Comment by praetoras** [comment]
Only the elevator modules, yeah: I had taken a stack of 9 elevator button modules and linked the entire stack to the root elevator as I had done before on the working elevators, then installed each of them in the screens (the elevator is the only thing they were being used for in this case).

Typically what I had done was "configure" one button, mine the screen, and then place the screen multiple times in creative to get a stack of identically configured elevator modules, but I also tried the vanilla way of just linking the stack and then configuring each one, but both ways gave the same result.

**Comment by praetoras** [comment]
Also, I should have mentioned this prior - I'm on mcjtylib-1.10-1.9.3 and rftools-1.10-4.67

**Comment by praetoras** [comment]
Hopefully this'll help:

I created a really simple test rig to replicate it: http://imgur.com/a/Bi7tc

I basically put out 7 elevators, powering the bottom elevator, with 10 blocks of red wool between each elevator level, and 1 screen controller between each two (except the 7th because that needed its own). The bottom 6 screens all showed all 7 levels, but the top 7th screen was blank as predicted. I didn't bother naming the levels or anything. I just linked the modules to the bottom elevator and stuck them in, then changed them to large buttons so they'd be easy to see against the red wool.

**Comment by McJty** [comment]
What other mods do you have? Also do you still have this problem with latest mcjtylib + rftools?

**Comment by sco0738** [comment]
I've been having the same problem after making an elevator from my base down to my mine at level 11. The elevator for the three levels of my base worked fine, but as soon as I tried extending it all the way down to my mine, everything stopped working. I even tried destroying everything and rebuilding the elevator system from scratch with no luck (after your latest update).

Also, I _do_ have an extra screen controller to include the level 11 screens. What is odd is that those screens are the only ones that work (but only displaying the mine level); the ones up near my base are the ones that are blank. Something else that is weird is that WAILA says that the working screens in the mine are not connected, not sure if this is a problem with WAILA or how it displayed back when the screens were working. Also, the screen controller in my base says it is connected to the screens, even though they no longer work. All screens I'm using right now have only the elevator module in them.

I'm using the FTB Unstable Pack with the latest versions of mcjtylib, rftools and dimentions mod that you released just a few hours ago.

Thanks for all your hard work. Your elevators are awesome, so I hope you don't have much trouble fixing this bug.

**Comment by praetoras** [comment]
Sorry for the long delay! I had to go on some trips that kept me away from the computer for a while. I am still experiencing the same issue (I'm not on the absolute latest release, but unless you've made changes specific to this issue it's likely still present at the moment. I updated RFTools not long ago). The mods I have are:

(It'll be easier to link you the pack): http://minecraft.curseforge.com/projects/star-fall-craft/relations/dependencies
